docs/07.Reference/7.05.Stream.md

```diff
@@ -16,10 +16,10 @@ whether a request/response is a stream, for example:
 
 * We can set `clientMaxBodySize` of an HTTP server to a negative value to
   tell Easegress the request is a stream, and not a stream otherwise. Please
-  refer [HTTP Server](./controllers.md#httpserver) for more information.
+  refer [HTTP Server](7.01.Controllers.md#httpserver) for more information.
```

Comment:
Should this file lay in the developer guide part instead of reference?

Reply:
The `Stream.md` is mainly used to explain `clientMaxBodySize` argument in `HTTPServer` and `serverMaxBodySize` argument in `Proxy`. It is referenced by `HTTPServer` and `Proxy` filter too.